omod/src/main/webapp/resources/htmlforms/sierra_leone/section-exam.xml

```diff
@@ -740,7 +763,7 @@
 
         <p/>
 
-        <div class="section-container-color">
+     <!-->   <div class="section-container-color">
```

Comment:
This might work, but html comment format is normally this:

`<!--  fskfjklsdjflkjf fkjfksdjfklsdjflksjfl -->`

not
`<!-->`